.env

```diff
@@ -0,0 +1,9 @@
+DOCKERHUB_NAME=username
+IMAGE_NAME=docker-postgres
+IMAGE_VERSION=0.0.1
+
+POSTGRES_HOST_PORT=5432
+POSTGRES_CONTAINER_PORT=5432
+POSTGRES_USER=username
```

Comment:
the user you should be creating is has_many_user and the db you should be creating is has_many_blogs

Reply:
oh.. I wasn't supposed to push this file up, too! -__- I fixed it though.

Reply:
This file is still in the branch and you're .gitignore file is not. Could that be the problem?